docs/config.rst

```diff
@@ -404,6 +404,14 @@ Complete list of settings that you can put into ``testenv*`` sections:
         setenv   =
             PYTHONPATH = {env:PYTHONPATH}{:}{toxinidir}
 
+    .. versionadded:: 3.20
+
+    Support for comments. Lines starting with ``#`` are ignored.
+
+    Support for environment files. Lines starting with the ``file|`` contain path to a environment
```

Comment:
@gaborbernat any specific reason you implemented it with a prefix vs a substitution? If I were to design this, I'd probably go for `{dotenv:file_path}` there... I'm not judging, it just seems untypical for tox's UX.

Reply:
Didn't want to worry about conflicts with conditional factors 🤷‍♂️

Reply:
And it's not really substitution at all happening here, but different load mechanism. Substitution implies we just string replace that expression with content of the file, but actually here we never do that. Conceptually.